properties.schema

```diff
@@ -45,10 +55,28 @@
                       "title": "Logo image alternative Text",
                       "inputType": "Text",
                       "validators": [],
-                      "help": "The alternative text for this image",
+                      "help": "The alternative text for this image.",
```

Comment:
I'd instead mention something about screenreaders here – at the moment it's just a repeat of the title.

Reply:
Thanks @tomgreenfield, a bit wordy but how about the following.

Provide alternative text for the image, such as your company name. This text is not a visible element. It is utilized by assistive technology such as screen readers. Avoid identifying the logo as actually being a logo.

Reply:
🤯 superb.